doc/source/index.rst

```diff
@@ -314,7 +314,6 @@ Papers
    raysgd/raysgd.rst
    raysgd/raysgd_pytorch.rst
    raysgd/raysgd_tensorflow.rst
-   raysgd/raysgd_dataset.rst
```

Comment:
Would it be better to not completely remove this yet, but instead just have warning box saying this is going to be deprecated in 1.7 and it is recommended to use Ray Datasets instead? When we actually do the full deprecation, then we could remove it from the docs.

Reply:
Hmm, I think it's ok to orphan it. After all, previous versions of the docs are kept up right?